### ISSUE TITLE
Change default history configurations

### ISSUE THREAD

**Issue by HarelM** [hluk/CopyQ#843]
Would be great if the following were checked by default after installation:
 * Focus Last window
 * Paste to current window
![image](https://user-images.githubusercontent.com/3269297/34160463-c4b9376a-e4d5-11e7-8ab9-647605a6fa98.png)

Also Ctrl+Shift+V for global action to toggle show/hide copyQ would improve usability IMHO, but that might be because I'm used to clipX.
Keep up the good work!

**Comment by hluk** [comment]
Few other people pointed out that "Paste to current window" should be enabled by default.

I use <kbd>Shift</kbd>+<kbd>Enter</kbd> shortcut to paste items and would be surprised if it happened on <kbd>Enter</kbd> key.

It's disabled by default because I believe it could lead to some unexpected and potentially dangerous behavior (e.g. pasting to console).

"Focus last window" is only useful if "Paste to current window" and "Close main window" are disabled. Maybe this option can be removed and at least renamed ("Unfocus/Lower main window").

**Comment by HarelM** [comment]
I'm not sure I follow - What should <kbd>Enter</kbd> do when the window is open and I want to paste the code to the last window I visited then?

Let's ignore all the stuff I wrote for a second and I'll describe the user scenario which I think is relevant to my use-case and also probably to other users:
1. I download, install and run CopyQ.
2. I copy some items.
3. I want to paste to the current location (just for this example - the chrome browser address bar) a clipboard item which isn't at the top of the current list.

As far as I understand the defaults after installation are not configured for this scenario using just the keyborad, right?
from what I gathered the equivalent with the mouse is to:
1. click the taskbar.
2. open CopyQ.
3. drag the relevant item from the list to the desired location.

As far as I'm concerned, as a user, this utility is aimed for productivity and so good keyboard shortcuts are essential.

If anything I wrote is incorrect I'll be happy to be wrong :-)
